webui/react/src/components/Markdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { Tabs } from 'antd';
+import { default as MarkdownViewer } from 'markdown-to-jsx';
```

Comment:
**question:** can't this just be:
```
import MarkdownViewer from 'markdown-to-jsx';
```

Reply:
Unfortunately not, since the default export from `markdown-to-JSX` is also just named `Markdown`, it causes a name collision with the component.